x-pack/plugins/security_solution/public/resolver/store/middleware.ts

```diff
@@ -77,15 +77,19 @@ export const resolverMiddlewareFactory: MiddlewareFactory = (context) => {
           }
           const nodeStats: Map<string, ResolverNodeStats> = new Map();
           nodeStats.set(entityId, stats);
+          const limitReached = children.nextChild !== null || ancestry.nextAncestor !== null;
```

Comment:
@jonathan-buttner does this seem right to you? If either nextChild or nextAncestor isn't `null` we know we didn't get all the stuff, right? Is there a similar thing for `lifecycle`?

Reply:
Ah my bad @bkimmel totally missed this. So the thing to note is that we're only requesting the default amount for the API. I wonder if we should always request the max?

As for `lifecycle` in the backend we're operating under the assumption that we'll always be able to retrieve all the lifecycle nodes for the entire request. We may have to address fixing that later though but for now there's no `nextLifecycle`.